pyomo/contrib/gdpopt/config_options.py

```diff
@@ -32,13 +56,10 @@ def _get_GDPopt_config():
         default=600,
         domain=PositiveInt,
         description="Time limit (seconds, default=600)",
-        doc="Seconds allowed until terminated. Note that the time limit can "
-            "currently only be enforced between subsolver invocations. You may "
-            "need to set subsolver time limits as well."
-    ))
-    CONFIG.declare("strategy", ConfigValue(
-        default=None, domain=In(_supported_strategies),
-        description="Decomposition strategy to use."
+        doc="""
+        Seconds allowed until terminated. Note that the time limit can
+        currently only be enforced between subsolver invocations. You may
+        need to set subsolver time limits as well."""
```

Comment:
This does not seem right. Usually because subsolver calls can be really long. In mindtpy we keep computing remaining times that are passed down to subsolvers.

Reply:
Good point, I will make that change.

Reply:
Sorry, @bernalde, I wasn't thinking very clearly the other day: How do you do this when you don't know what solver is being used? Could you point me to where that happens in mindtpy?

Reply:
It is actually a GDPOpt util https://github.com/Pyomo/pyomo/blob/6ecb587581bd64bc5756b71662c7a66b1211632a/pyomo/contrib/gdpopt/util.py#L517

Reply:
Yes, but I couldn't find in Mindtpy where the time remaining is passed to the subsolver. I'm unsure (short of a huge string of `if` statements) of how to do this in a general way because the option names are solver-specific. For example, the option is `timelimit` in cplex but `tmlim` in glpk, I think.

Reply:
You are right! We have discussed this briefly with @jsiirola. Whether it makes sense to have an alias for the important options for all solvers, but for the time being, I guess that we cannot consider the remaining time and just hope for the best with each suproblem.